cheatsheets/JSON_Web_Token_Cheat_Sheet_for_Java.md

```diff
@@ -455,7 +455,7 @@ String token = tokenCipher.decipherToken(cipheredToken, this.keyCiphering);
 
 ### Symptom
 
-It's occur when a application store the token in a way allowing this one to be:
+This occurs when a application stores the token in a manner exibiting the following behavior:
```

Comment:
After I have passed the sentence to Antidode (English UK) and applied the proposed fixes it will become:

> This occurs when an application stores the token in a manner exhibiting the following behaviour